doc/source/dev/howto_build_docs.rst

```diff
@@ -58,18 +58,28 @@ new virtual environment is recommended.
 Dependencies
 ^^^^^^^^^^^^
 
-All of the necessary dependencies for building the NumPy docs can be installed
-with::
+All of the necessary dependencies for building the NumPy docs except for
+Doxygen_ can be installed with::
 
     pip install -r doc_requirements.txt
 
-We currently use Sphinx_ for generating the API and reference
-documentation for NumPy. In addition, building the documentation requires
-the Sphinx extension `plot_directive`, which is shipped with
+We currently use Sphinx_ along with Doxygen_ for generating the API and
+reference documentation for NumPy. In addition, building the documentation
+requires the Sphinx extension `plot_directive`, which is shipped with
```

Comment:
Since `make html` will still work without it; consider splitting into two sentences.

```
We currently use Spinx_ to generate most of the API documentation. However, the C-Numpy API documentation is generated with Doxygen_.
```

Reply:
Maybe there is some misunderstanding. Doxygen will not be used alone to generate the C-API documentation, but perhaps we can revisit this in the future. This PR implements using Doxygen to pull formatted comments out of the C code and turn it into XML, then uses breeze to turn that XML into reST, then uses Sphinx to turn the reST into HTML/PDF/Latex. So the second proposed sentence is less correct than the original.

Reply:
From the end users point of view, isn't the current implementation expected to throw warnings for the C-API sections of the documentation and otherwise work as is? From this perspective it makes sense to explain the change in a minimally invasive manner here while leaving the details:
Comments->XML->RST->Combined docs
can be elsewhere. No strong feelings about this though.

Reply:
**3. Inclusion directives** is pretty clear I guess:

> Breathe_ provides a wide range of custom directives to allow
converting the documents generated by Doxygen_ into reST files.

Do you think it isn't clear enough? the reader should check Breathe docs after all.